src/test/java/com/google/cloud/solutions/autotokenize/common/TransformingReaderTest.java

```diff
@@ -60,7 +60,7 @@
 
     @Rule public transient TestPipeline testPipeline = TestPipeline.create();
 
-    private static transient DB testDatabase;
+    private static MySQLContainer databaseContainer;
```

Comment:
Any specific reason for not making the testContainer object transient?

Reply:
Since static fields are excluded from serialization, may I ask out of curiosity why you are marking them as transient? :)